app/channel.go

```diff
@@ -596,10 +596,48 @@ func (a *App) postChannelPrivacyMessage(user *model.User, channel *model.Channel
 	return nil
 }
 
-func (a *App) RestoreChannel(channel *model.Channel) (*model.Channel, *model.AppError) {
+func (a *App) RestoreChannel(channel *model.Channel, userId string) (*model.Channel, *model.AppError) {
+	var user *model.User
+	if userId != "" {
```

Comment:
I think we are fine to require `userId` to be ` != ""` and drop this check.

Reply:
not sure what you mean. but are you saying we should just drop this if statement?

Reply:
Yes